specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
At some point it would seem useful to  distinguish between a `csv` (the whole file: headers defining columns, rows) and a `csv_row` (a single row). Or perhaps that isn't needed, if we spec this as a conversion from a sequence of cvrs to a csv.
`  as_csv: [ sequence[cvr] -> csv ]`